src/sinks/http.rs

```diff
@@ -336,6 +358,21 @@ mod tests {
         assert_eq!(output.message, "hello world".to_string());
     }
 
+    #[test]
+    fn http_encode_event_json_metrics_plus_mode() {
+        let encoding = EncodingConfig::from(Encoding::Ndjson);
+        let event = Event::from("hello world");
```

Comment:
I think this is just creating a `Event::Log` event. We probably want to test a `Event::Metric` here. You can create one directly with `Event::Metric(Metric{ ... })`.